model-optimizer/mo/utils/cli_parser.py

```diff
@@ -1234,12 +1234,15 @@ def check_positive(value):
     return int_value
 
 
-def depersonalize(value: str):
+def depersonalize(value: str, key: str):
+    dir_keys = [
+        'output_dir', 'o', 'extensions', 'saved_model_dir', 'tensorboard_logdir', 'caffe_parser_path'
```

Comment:
Is it necessary to add "o"  key? Parser should save it as "output_dir" if "o" is specified.

Reply:
You are right. It is not necessary, removed it